.codesandbox/ci.json

```diff
@@ -6,6 +6,6 @@
   "publishDirectory": {
     "@base_ui/react": "packages/mui-base/build"
   },
-  "sandboxes": ["/examples/base-ui-cra-ts"],
+  "sandboxes": ["react-vite-ts"],
```

Comment:
This doesn't really make sense, as `react-vite-ts` doesn't depend on `@base_ui/react`. I'd ask the team if they would find the sandboxes generated per PR useful, and if yes, I'd leave just this one directory in /examples. Otherwise, we can remove the `sandboxes`.

Reply:
I think they are going to be useful, yeah. I hadn't used CodeSandbox CI before, but from reading the description it sounds like something that would be good to set up once we go a bit more public.

Thoughts on putting the template it `/.codesandbox/template/`?
No big deal but `/examples/` sets up the expectation to see something that resembles a less or more complete app built with Base UI

Reply:
I have never used this `"sandboxes"` feature of CodeSandbox CI 

<img width="382" alt="SCR-20241021-cddi" src="https://github.com/user-attachments/assets/115ab656-7409-4fef-8d88-4b555df40618">

https://ci.codesandbox.io/status/mui/base-ui/pr/744/builds/552351

I suspect that we can delete this. Starting from https://deploy-preview-744--base-ui.netlify.app/components/react-field/ feels much more convenient:

<img width="241" alt="SCR-20241021-cdgh" src="https://github.com/user-attachments/assets/91decd47-7e45-40c3-907e-527bcc36ddb3">